### PR TITLE
W-12559775 config HA (multiple replicas, horizontal scaling) duke

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+[%header,cols="1,2a"]
+|===
+| Parameter | Description
+| `autoscaling.enabled` | Boolean that indicates whether the Horizontal Pod Autoscaler (HPA) is enabled. Defaults to `false`.
+| `autoscaling.minReplicas` | The minimum number of replicas that the HPA scaler is allowed to create. Defaults to `2`.
+| `autoscaling.maxReplicas` | The maximum number of replicas that the HPA scaler is allowed to create. Defaults to `11`.
+| `autoscaling.targetCPUUtilizationPercentage` | A resource metric that sets the average CPU usage percentage of all deployed Pods. Defaults to `50`.
+| `autoscaling.targetMemoryUtilizationPercentage` | A string representing the average memory usage percentage of all deployed Pods. Defaults to `nil`.
+| `autoscaling.extraMetrics` | A list of custom Pod configuration metrics that enables you to collect and monitor application-specific metric data. Defaults to `[]`. Define the metrics with nested parameters, for example:
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: from Israel, get data with commands like this:
kubectl get --raw /apis/metrics.k8s.io/v1beta1/nodes/<NODE_NAME>
kubectl get --raw /apis/metrics.k8s.io/v1beta1/namespaces/<NAMESPACE>/pods/<POD_NAME>

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+ingress flex-gateway/flex-gateway \
+--set-file registration.content=registration.yaml \
+--set replicaCount=2
+----
++
+When successful, the command prints output similar to this example: 
++
+----
+Release "ingress" does not exist. Installing it now.
+NAME: ingress2
```
Comment (IsraelZeromski) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: according to command the correct output name is ingress

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+helm -n gateway upgrade \
+ingress flex-gateway/flex-gateway \
+--reuse-values \
+-f autoscaling.yaml
+----
++
+When successful, the command prints output similar to this example: 
++
+----
+LAST DEPLOYED: Tue Apr 18 15:46:38 2023
```
Comment (IsraelZeromski) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:182]: add output `NAME: ingress` field

**Review by IsraelZeromski — CHANGES_REQUESTED** [review]

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 2 above)
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
NAME: ingress
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:182]:
```suggestion
Release "ingress" has been upgraded. Happy Helming!
NAME: ingress
LAST DEPLOYED: Tue Apr 18 15:46:38 2023
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 3 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:182]: done

**Review by IsraelZeromski — APPROVED** [review]
look fine

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+== Configure Autoscaling
+
+Horizontal Pod autoscaling generates Flex Gateway replicas automatically based on CPU, memory, and other target and behavioral settings. 
+
+[[autoscaling-config]]
+=== Update Your Helm Chart with an Autoscaling Configuration
+
+Update your Helm chart with autoscaling parameters. 
+
+. Create a YAML file with your autoscaling configuration, modifying the example settings you require.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
. Create a YAML file with your autoscaling configuration, modifying the settings to meet your requirements.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+    scaleUp:
+      selectPolicy: Max
+      stabilizationWindowSeconds: 100 
+      policies:
+      - type: Percent
+        value: 70
+        periodSeconds: 30
+----
++
+For descriptions of these settings, see <<autoscaling-parameters>>.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
++
+----
+NAME                  DESIRED   CURRENT   READY   AGE
+ingress-5b7474b8f6    2         2         2       70s
+----
+
+[[autoscale]]
+== Configure Autoscaling
+
+Horizontal Pod autoscaling generates Flex Gateway replicas automatically based on CPU, memory, and other target and behavioral settings.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
Horizontal Pod autoscaling generates Flex Gateway replicas automatically based on CPU, memory, and other target and behavioral settings. For descriptions of these settings, see <<autoscaling-parameters>>. 
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+kubectl get hpa -n gateway
+----
++
+The command prints output similar to this example: 
++
+----
+NAME     REFERENCE           TARGETS          MINPODS  MAXPODS  REPLICAS  AGE
+ingress  Deployment/ingress  66%/50%, 5%/50%  2        11       2         59s
+----
+//TODO: TIME PERMITTING, IT WOULD BE GOOD TO ADD SOME TROUBLESHOOTING INFO
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:241]: editors: don't worry about this and any other todos on this beta doc

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+
+[[autoscaling-parameters]]
+=== Autoscaling Parameters
+
+To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other Helm chart settings to define autoscaling . Autoscaling parameters in the Helm chart follow.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other Helm chart settings to define autoscaling behavior. Autoscaling parameters in the Helm chart follow.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 10]
```diff
@@ -0,0 +1,286 @@
...
+
+----
+kubectl get --raw /apis/metrics.k8s.io/v1beta1/nodes/<node-name>
+
+kubectl get --raw /apis/metrics.k8s.io/v1beta1/namespaces/<namespace>/pods/<pod-name>
+----
+| `autoscaling.behavior` | A setting that supports the `HorizontalPodAutoscaler` (HPA) object in Kubernetes. HPA settings control autoscaling behavior, such as the number of application resources based on the rate of change to the workload. 
+| `autoscaling.behavior.scaleUp`, `autoscaling.behavior.scaleDown` | Settings for autoscaling behavior when the HPA scaler increases (`scaleUp`) or decreases (`scaleDown`) the number of replicas. Define scaling behavior with nested parameters: 
+
+* `selectPolicy`: Setting that determines when to scale Pods when a metric of configured resources indicates that scaling is require, which can contain different numbers of replicas. Valid values are `Disabled`, `Min`, and `Max`. Defaults to `Max`. The `Min` and `Max` values indicate whether to scale Pods with the lowest (`Min`) or highest (`Max`) number of replicas.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: this one is awkward. I need to reword... :(

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 11]
```diff
@@ -0,0 +1,286 @@
...
+* A Flex Gateway deployment or a registered Flex Gateway instance (gateway)
++
+For gateway registration and deployment processes, see xref:flex-gateway-set-up.adoc[] or xref:flex-gateway-k8-getting-started.adoc[]. 
+
+* A Kubernetes cluster for your Flex Gateway deployment
+
+[[replica-count]]
+== Set a Replica Count
+
+Configure and verify the number of replicas that you require. The steps to follow depend on whether a Helm chart is installed.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: require -> need?

I read "The steps to follow" as equal to "The following steps" not "The steps to take".

Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
Configure and verify the number of replicas that you require. The steps to take depend on whether a Helm chart is installed. 
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 12]
```diff
@@ -0,0 +1,286 @@
...
+* A Flex Gateway deployment or a registered Flex Gateway instance (gateway)
++
+For gateway registration and deployment processes, see xref:flex-gateway-set-up.adoc[] or xref:flex-gateway-k8-getting-started.adoc[]. 
+
+* A Kubernetes cluster for your Flex Gateway deployment
+
+[[replica-count]]
+== Set a Replica Count
+
+Configure and verify the number of replicas that you require. The steps to take depend on whether a Helm chart is installed.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:28]: changed to "to take" based on review

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 11 above)
Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: One idea...
. Depending on your configuration, run the following command:
** If a Helm chart is installed:
** If a Helm chart is not installed:

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 13]
```diff
@@ -0,0 +1,286 @@
...
+
+[[autoscale]]
+== Configure Autoscaling
+
+Horizontal Pod autoscaling generates Flex Gateway replicas automatically based on CPU, memory, and other target and behavioral settings. For descriptions of these settings, see <<autoscaling-parameters>>. 
+
+[[autoscaling-config]]
+=== Update Your Helm Chart with an Autoscaling Configuration
+
+Update your Helm chart with autoscaling parameters.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
To update your Helm chart with autoscaling parameters:
```

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: (or just remove the intro since it's nearly the same as the title)

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 11 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: I changed it to "The steps to take," which sounds better to me. Re the alternative, if they are installing for the first time, they don't have a config yet, so can't really say "Depending on your configuration."

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 14]
```diff
@@ -0,0 +1,286 @@
...
+    scaleUp:
+      selectPolicy: Max
+      stabilizationWindowSeconds: 100 
+      policies:
+      - type: Percent
+        value: 70
+        periodSeconds: 30
+----
+
+. If a Helm chart for your namespace is installed, run the following command to apply your autoscaling configuration:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:158]: same suggestion as before.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 15]
```diff
@@ -0,0 +1,286 @@
...
+----
+NAME     REFERENCE           TARGETS          MINPODS  MAXPODS  REPLICAS  AGE
+ingress  Deployment/ingress  66%/50%, 5%/50%  2        11       2         59s
+----
+//TODO: TIME PERMITTING, IT WOULD BE GOOD TO ADD SOME TROUBLESHOOTING INFO
+
+[[autoscaling-parameters]]
+=== Autoscaling Parameters
+
+To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other Helm chart settings to define autoscaling behavior. Autoscaling parameters in the Helm chart follow.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: Suggest removing second sentence and/or adding a title to the table "Autoscaling Parameters" or add "Autoscaling to the Parameter column title.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 16]
```diff
@@ -0,0 +1,286 @@
...
+
+[%header,cols="1a,2a"]
+|===
+| Parameter | Description
+| `autoscaling.enabled` | Boolean that indicates whether the Horizontal Pod Autoscaler (HPA) is enabled. Defaults to `false`.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:251]: You could have a Default Value column and put the default values there.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 17]
```diff
@@ -0,0 +1,286 @@
...
+| `autoscaling.minReplicas` | The minimum number of replicas that the HPA scaler is allowed to create. Defaults to `2`.
+| `autoscaling.maxReplicas` | The maximum number of replicas that the HPA scaler is allowed to create. Defaults to `11`.
+| `autoscaling.targetCPUUtilizationPercentage` | A resource metric that sets the average CPU usage percentage of all deployed Pods. Defaults to `50`.
+| `autoscaling.targetMemoryUtilizationPercentage` | A string representing the average memory usage percentage of all deployed Pods. Defaults to `nil`.
+| `autoscaling.extraMetrics` | A list of custom Pod configuration metrics that enables you to collect and monitor application-specific metric data. Defaults to `[]`. Define the metrics with nested parameters, for example:
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: Does the list enable you or the metrics? If list, it's fine. If metrics, "enable".

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 18]
```diff
@@ -0,0 +1,286 @@
...
+
+----
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:258]: This is a lot of stuff to have in the middle of the table. Suggest adding a small section below and linking to it from the table.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 19]
```diff
@@ -0,0 +1,286 @@
...
+kubectl get --raw /apis/metrics.k8s.io/v1beta1/nodes/<node-name>
+
+kubectl get --raw /apis/metrics.k8s.io/v1beta1/namespaces/<namespace>/pods/<pod-name>
+----
+| `autoscaling.behavior` | A setting that supports the `HorizontalPodAutoscaler` (HPA) object in Kubernetes. HPA settings control autoscaling behavior, such as the number of application resources based on the rate of change to the workload. 
+| `autoscaling.behavior.scaleUp`, `autoscaling.behavior.scaleDown` | Settings for autoscaling behavior when the HPA scaler increases (`scaleUp`) or decreases (`scaleDown`) the number of replicas. Define scaling behavior with nested parameters: 
+
+* `selectPolicy`: Setting that determines when to scale Pods when a metric of configured resources indicates that scaling is require, which can contain different numbers of replicas. Valid values are `Disabled`, `Min`, and `Max`. Defaults to `Max`. The `Min` and `Max` values indicate whether to scale Pods with the lowest (`Min`) or highest (`Max`) number of replicas.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
* `selectPolicy`: If a metric of configured resources indicates that scaling is require, this setting determines when to scale Pods. This setting is necessary because Pods can contain different numbers of replicas. Valid values are `Disabled`, `Min`, and `Max`. Defaults to `Max`. The `Min` and `Max` values indicate whether to scale Pods with the lowest (`Min`) or highest (`Max`) number of replicas.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 10 above)
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
* `selectPolicy`: Setting that determines when to scale Pods when a metric of configured resources indicates that scaling is required, which can contain different numbers of replicas. Valid values are `Disabled`, `Min`, and `Max`. Defaults to `Max`. The `Min` and `Max` values indicate whether to scale Pods with the lowest (`Min`) or highest (`Max`) number of replicas.
```

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: The first sentence is a little hard to follow.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 16 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:251]: I had that, but some cells would be empty. It's not easy to specify defaults for those. Can't say "none" with confidence. The workaround is to omit that column.

**Review by hannanelson — APPROVED** [review]

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 17 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: Hmm, the list (as in a YAML array/list) of settings is applied to the Pod, and if the Pod runs with those settings, then you'll be able to get the data. Let me try to rework it a bit.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 18 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:258]: An issue is that there's nothing more I can say about those settings, so I think the context here gives those settings more meaning.

Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:258]: I'm hoping the field will give me more to say about those settings though, and then I will probably turn them into bullets with descriptions.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 20]
```diff
@@ -0,0 +1,286 @@
...
+
+[%header,cols="1a,2a"]
+|===
+| Parameter | Description
+| `autoscaling.enabled` | Boolean that indicates whether the Horizontal Pod Autoscaler (HPA) is enabled. Defaults to `false`.
+| `autoscaling.minReplicas` | The minimum number of replicas that the HPA scaler is allowed to create. Defaults to `2`.
+| `autoscaling.maxReplicas` | The maximum number of replicas that the HPA scaler is allowed to create. Defaults to `11`.
+| `autoscaling.targetCPUUtilizationPercentage` | A resource metric that sets the average CPU usage percentage of all deployed Pods. Defaults to `50`.
+| `autoscaling.targetMemoryUtilizationPercentage` | A string representing the average memory usage percentage of all deployed Pods. Defaults to `nil`.
+| `autoscaling.extraMetrics` | A list of custom Pod configuration metrics that enables you to collect and monitor application-specific metric data. Defaults to `[]`. Define the metrics with nested parameters, for example:
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
| `autoscaling.extraMetrics` | A list of custom Pod configuration metrics. When a Pod runs with these settings, you can collect and monitor application-specific metric data. Defaults to `[]`. Define the metrics with nested parameters, for example:
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 21]
```diff
@@ -0,0 +1,286 @@
...
+----
+NAME     REFERENCE           TARGETS          MINPODS  MAXPODS  REPLICAS  AGE
+ingress  Deployment/ingress  66%/50%, 5%/50%  2        11       2         59s
+----
+//TODO: TIME PERMITTING, IT WOULD BE GOOD TO ADD SOME TROUBLESHOOTING INFO
+
+[[autoscaling-parameters]]
+=== Autoscaling Parameters
+
+To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other Helm chart settings to define autoscaling behavior. Autoscaling parameters in the Helm chart follow.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other Helm chart settings to define autoscaling behavior.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 10 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: done

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 14 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:158]: see comment on convo page about "configuration'

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 15 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: done

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 17 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: revised

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 22]
```diff
@@ -0,0 +1,286 @@
...
+
+----
+kubectl get --raw /apis/metrics.k8s.io/v1beta1/nodes/<node-name>
+
+kubectl get --raw /apis/metrics.k8s.io/v1beta1/namespaces/<namespace>/pods/<pod-name>
+----
+| `autoscaling.behavior` | A setting that supports the `HorizontalPodAutoscaler` (HPA) object in Kubernetes. HPA settings control autoscaling behavior, such as the number of application resources based on the rate of change to the workload. 
+| `autoscaling.behavior.scaleUp`, `autoscaling.behavior.scaleDown` | Settings for autoscaling behavior when the HPA scaler increases (`scaleUp`) or decreases (`scaleDown`) the number of replicas. Define scaling behavior with nested parameters: 
+
+* `selectPolicy`: If a metric of configured resources indicates that scaling is require, this setting determines when to scale Pods. This setting is necessary because Pods can contain different numbers of replicas. Valid values are `Disabled`, `Min`, and `Max`. Defaults to `Max`. The `Min` and `Max` values indicate whether to scale Pods with the lowest (`Min`) or highest (`Max`) number of replicas.
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]:
```suggestion
* `selectPolicy`: If a metric of configured resources indicates that scaling is required, this setting determines when to scale Pods. This setting is necessary because Pods can contain different numbers of replicas. Valid values are `Disabled`, `Min`, and `Max`. Defaults to `Max`. The `Min` and `Max` values indicate whether to scale Pods with the lowest (`Min`) or highest (`Max`) number of replicas.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 10 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: I broke up that sentence. Hoping it's more easy to follow now.

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 11 above)
Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: The reason I suggest the bullet list approach is that steps imply linear (do this, then this) and bullets imply options. (At least, to me.)

Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc]: Yeah, I started that way, but then I decided that the space between the bullets made it less easy to use. There were some other issues that I don't remember off the top of my head right now. But I'm usually thinking about stuff like this when I write docs.
